### ISSUE TITLE
No man page for pg_dump

### ISSUE THREAD

**Issue by RobertFischer** [NixOS/nixpkgs#25928]
## Issue description

When I install the postgresql service, I don't have `man pg_dump`.

### Steps to reproduce
In your config:
`service.postgresql.enable = true`

Rebuild.

Verify that the `pg_dump` executable does exist.  (It should.)

From the shell, `man pg_dump` reports no man page found.

## Technical details

* Nix version: 17.03

**Comment by emmanuelrosa** [comment]
To get `pg_dump` and friends you can install the `postgresql` package; along with the `service.postgresql` you already have; `service.postgresql` uses `postgresql`.

As for the man page, there is a derivation for it (*[hash]-postgresql-x.y.x-doc*) but I don't know why `man` doesn't see it.

```
find /nix/store/x66g4f8v3vm1l5wd2gz6bj56w2h2vr3c-postgresql-9.6.2-doc/share/ -name pg_dump*
/nix/store/x66g4f8v3vm1l5wd2gz6bj56w2h2vr3c-postgresql-9.6.2-doc/share/man/man1/pg_dump.1.gz
/nix/store/x66g4f8v3vm1l5wd2gz6bj56w2h2vr3c-postgresql-9.6.2-doc/share/man/man1/pg_dumpall.1.gz
```

**Comment by Mic92** [comment]
This is fixed now in 029125b973e4c44b20e288d43dfb9a46b18e3ef7. However I would like not to backport this at the moment, as it moves some files from one output to another, which might break other peoples setup. To get manpages in 17.03 or 17.09, just add `postgresql.doc` to `systemPackages`.
